task03/task03/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task03
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int x;
+            x = int.Parse(Console.ReadLine());
```

Comment:
Можно было записать одни  оператором
```suggestion
            int x = int.Parse(Console.ReadLine());
```